### PR TITLE
#41 add sanitizeStylisedHtml filter to inbox libs

### PR DESCRIPTION
**Pull request by asmaNode**

### CODE REVIEW

File: src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+
+(function(angular) {
```
Comment (tuanlc) [inline src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js:3]: remove IIFE

File: src/linagora.esn.unifiedinbox/app/app.js [hunk 2]
```diff
@@ -205,7 +205,6 @@ require ('./filters.js');
 require ('./filters/filter-descendant-mailboxes.js');
 require ('./filters/item-date.js');
 require ('./filters/quote/quote.js');
-require ('./filters/sanitize-stylised-html-filter.js');
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/app/app.js:208]: You need to remove the file also

**Comment by huy-ta** [comment]
You forgot to link the issue to the PR. You can link them by adding this in your PR's description: "Resolves _link to the issue_" or "Fixes _link to the issue_".

**Review by huy-ta — APPROVED** [review]

File: src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const _ = require('lodash');
+
+/* global sanitizeHtml: false */
+angular.module('esn.inbox.libs').filter('sanitizeStylisedHtml', function ($sce) {
```
Comment (piavgh) [inline src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js:6]: The variable `$sce` is ambiguous, I don't understand its purpose just by looking at the name.
Can you please use a longer name to make it more clear?

**Review by rezk2ll — APPROVED** [review]

File: src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js (hunk 3 above)
Comment (huy-ta) [reply src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js:6]: It's not a variable that we define. It belongs to AngularJS, like `$q`.

Comment (asmaNode) [reply src/esn.inbox.libs/app/filter/sanitize-stylised-html-filter.js:6]: i just replaced the file from unified inbox to inbox libs ;)

**Pull request merged**